src/v3/src/transformer/terminal/transformTerminalTransaction.ts

```diff
@@ -171,6 +171,20 @@ export const transformTerminalTransaction = (
     );
   }
 
+  if (transaction.context?.failure?.href) {
```

Comment:
Can we add tests to cover this scenario?

Reply:
would an integration test be necessary?  It would essentially test the same thing as the testcafe test.  Or do you mean in the unit test for `transformTerminalTransaction`?

Reply:
I was thinking of the unit tests for `transformTerminalTransaction`

Reply:
ah yes okay for sure I will add that 👍